src/main/java/org/apache/maven/shared/dependency/analyzer/DefaultProjectDependencyAnalyzer.java

```diff
@@ -94,7 +97,7 @@ public ProjectDependencyAnalysis analyze( MavenProject project )
             Set<Artifact> testArtifactsWithNonTestScope = getTestArtifactsWithNonTestScope( testOnlyArtifacts );
 
             return new ProjectDependencyAnalysis( usedDeclaredArtifacts, usedUndeclaredArtifactsWithClasses,
-                                                  unusedDeclaredArtifacts, testArtifactsWithNonTestScope );
+                unusedDeclaredArtifacts, testArtifactsWithNonTestScope );
```

Comment:
Contradicts our style

Reply:
multiline arguments should be aligned for method call - ok

There are no examles / rules ... https://maven.apache.org/developers/conventions/code.html

Reply:
It's incomplete. What I have seen in Core and other components is to align on the same column